content/news/translation/2023/do-you-speak-prestashop-december-2022.md

```diff
@@ -0,0 +1,151 @@
+|-|---------- | -------- | ----------------
+| 1. | Ramon J. Batlle (rjbatllet) | Catalan | 1,379
+| 2. | Rick Gzz (rickygzz85) | Spanish, Mexico | 780
+| 3. | SNor42 | Hungarian | 680
+| 4. | hienvuneu| Vietnamese | 165
+| 5. | danielshapiro9 | Hebrew | 155
+
+#### Top 3 proofreaders in December 2022:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | SNor42 | Hungarian | 972
+| 2. | danielshapiro9 | Hebrew | 161
+| 3. | M. Mirena (mmirena) | Albanian | 58
+
+Congratulations, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of December, PrestaShop 8 was fully available (= 100% translated and validated) in **11 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Catalan | Chinese (CN) | **Chinese (TW)** | Czech |
+| Dutch | French | Greek | Italian | 
+| Portuguese (BR) | Spanish (ES) | Ukrainian | 
+
+Thank you all for your contribution! :muscle: 
+
+#### Almost there!
+
+Korean, Lithuanian, Romanian, and Polish are almost fully available. There is only 1% of strings to translate and proofread! Keep up the great work.
+
+## Languages with the best evolution
+
+The following languages had the best progress in December thanks to the translation community:
+ 
+* **Persian** (+4% to reach 97% translated and 97% validated)
+* **Hungarian** (+4% to reach 96% translated and 96% validated) 
+* **Romanian** (+1% to reach 100% translated and 96% validated)
+* **Chinese (TW)** (+1% to reach 100% translated and 100% validated)
+
+
+Congratulations! 
+
+![Best translation progress in December 2022](/assets/images/2023/01/build-crowdin-progress-december22.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
+
+## Let’s keep in touch!
+
+If you have not joined us on Crowdin yet, [it is never too late](https://crowdin.com/project/prestashop-official)! :wink:
+
+You can also join the #translation channel of the PrestaShop Open Source Slack. On this channel, I share information about the translation project. You will also be informed about the publication of the Crowdin monthly report. Of course, you will also be able to ask questions, suggest ideas, or get in touch with other translators.
+
+Thanks everyone! 🙌
+
+{{< cta "https://join.slack.com/t/prestashop/shared_invite/zt-dkmbz5qf-I~FlEWwmRUOXunc5ui0Ucg" >}}Join us on Slack{{< /cta >}}
```

Comment:
```suggestion
{{< cta "https://www.prestashop-project.org/slack/" >}}Join us on Slack{{< /cta >}}
```

Reply:
Hello, this is the direct link to the translation channel :) Should I update it?

Reply:
This invitation link is no longer valid and requires login via google/apple or @prestashop.com email